packages/components/src/components/Link/Link.tsx

```diff
@@ -241,7 +243,7 @@ export const Link: FC<LinkProps> = ({
   const iconDisplay =
     _icon === 'no icon' ? null : (
       <View style={iconViewStyle}>
-        <Icon fill={linkColor} {..._icon} />
+        <Icon fill={linkColor} {..._icon} width={50} height={50} />
```

Comment:
Remove testing code.
```suggestion
        <Icon fill={linkColor} {..._icon} />
```